src/transformers/pipelines.py

```diff
@@ -954,7 +958,11 @@ def __init__(self, return_all_scores: bool = False, **kwargs):
             else MODEL_FOR_SEQUENCE_CLASSIFICATION_MAPPING
         )
 
+        if hasattr(self.model.config, "return_raw_outputs") and return_raw_outputs is None:
+            return_raw_outputs = self.model.config.return_raw_outputs
+
         self.return_all_scores = return_all_scores
+        self.return_raw_outputs = return_raw_outputs if return_raw_outputs is not None else False
```

Comment:
Looks good, but we probably should be more consistent in our way of managing `pipe arguments` vs `call arguments`....